secv_guis/bimask_app/main_window.py

```diff
@@ -776,7 +780,7 @@ def _handle_img_selection(self, basename):
         abspath = os.path.join(self.file_lists.img_list.dirpath, basename)
         success = self.graphics_view.new_image(abspath, self.mask_color,
                                                self.preannot_color)
-        if self.file_lists.preannot_list is not None:
+        if self.file_lists.preannot_list is not None:                                       
```

Comment:
Linting: are you adding whitespaces after the colon and before the newline here? ideally no empty whitespaces after the code in a line

Reply:
I'm gonna clean the files with next PR